debsources/app/copyright/routes.py

```diff
@@ -35,6 +36,16 @@ def skeleton_variables():
     lambda e: (ErrorHandler()(e, http=404), 404))
 
 
+# Before request
+@bp_copyright.before_request
+def before_request():
+    endpoints = ['license', 'file']
+    if any(endpoint in request.endpoint for endpoint in endpoints):
```

Comment:
`request.endpoint` is a string. I think what you want is
`if request.endpoint in endpoints`
Right?